eng/common/scripts/logging.ps1

```diff
@@ -31,10 +31,10 @@ function LogDebug
 {
     if ($isDevOpsRun) 
     {
-        Write-Host "##vso[task.LogIssue type=debug;]$args"
+        Write-Host "[debug]$args"
```

Comment:
I'd still prefer to keep this as it was.

Reply:
Sorry this is correct after looking through the docs. https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#formatting-commands